transform/javassist/src/internal/java/net/flintmc/transform/javassist/internal/DefaultClassTransformService.java

```diff
@@ -75,6 +76,10 @@ private DefaultClassTransformService(
     this.consumerBasedClassTransformMetaFactory = consumerBasedClassTransformMetaFactory;
     this.classMappingProvider = classMappingProvider;
     this.classTransformMetas = new ArrayList<>();
+
+    String rawPrefixes = System.getProperty(
+        "net.flintmc.transform.prefix", "net.minecraft;com.mojang");
+    this.classPrefixes = rawPrefixes.isEmpty() ? new String[0] : rawPrefixes.split(";");
```

Comment:
Can we replace this `;` with `,`?
On linux this could cause confusion:
```
$> java -jar -Dnet.flintmc.transform.prefix=my.company;other.domain example.jar
bash: command not found: other.domain
```
Linux shells interpret `;`as a command separator